display/state.py

```diff
@@ -32,6 +33,18 @@ def playGame(screen, keyboard):
         human = Human()
         sprites.add(human)
 
+        i_like_this = Projectile('like', -6, 250, 2)
+        sprites.add(i_like_this)
+
+        b = Projectile('wa', 4, 117, 20)
+        sprites.add(b)
```

Comment:
Beep beep I'm a sheep
I said beep beep I'm a sheep
Beep beep I'm a sheep
I said beep beep I'm a sheep
BEEP BEEP I'm a sheep
I said beep beep I'm a sheep
BEEP BEEP I'm a sheep
I said beep beep I'm a sheep
Yeah, now some of you might be wondering
How exactly does one beep beep like a sheep?
Take it away, Gabe!
Step one
Throw your hands up
And point them to the floor
Step two
Here's what to do
Now get down on all fours
Step three
Just bounce around
It's easy, follow me
Step four
Go crazy now!
And beep beep like a sheep!
Beep beep I'm a sheep
I said beep beep I'm a sheep
Beep beep I'm a sheep
I said beep beep I'm a sheep
BEEP BEEP I'm a sheep
I said beep beep I'm a sheep
BEEP BEEP I'm a sheep
I said beep beep I'm a sheep
Beep beep!
You've got to
You've got to
Beep beep
I said Beep Beep like a sheep!
Yeah, so you know how to beep beep like a sheep, I see
But is that all you can do?
No! No! No! No!
Step one
Throw your hands up
And point them to the sky
Step two
Drop to the floor
And move from side to side
Step three
Just bounce around
C'mon I'll show you how
Step four
Go crazy now
And meow meow like a cow!
Meow meow I'm a cow
I said meow meow I'm a cow
Meow meow I'm a cow
I said meow meow I'm a cow
Meow meow I'm a cow
I said meow meow I'm a cow
Meow meow I'm a cow
I said meow meow I'm a cow
Beep beep
You've got to
You've got to
Beep beep
I said beep beep I'm a sheep
Beep beep
You've got to
You've got to
Beep beep
I said beep beep like a sheep!
Beep beep